packages/mobile/src/components/now-playing-drawer/AudioControls.tsx

```diff
@@ -69,6 +77,28 @@ export const AudioControls = () => {
     }
   }, [isPlaying, dispatch])
 
+  const onPressShuffle = useCallback(() => {
+    if (shuffleEnabled) {
+      dispatchWeb(shuffle({ enable: false }))
+    } else {
+      dispatchWeb(shuffle({ enable: true }))
+    }
+  }, [dispatchWeb, shuffleEnabled])
+
+  const onPressRepeat = useCallback(() => {
+    switch (repeatMode) {
+      case RepeatMode.ALL:
+        dispatchWeb(repeat({ mode: RepeatMode.SINGLE }))
```

Comment:
This is probably more expressive but I'm not a huge fan of calling dispatchWeb(repeat()) in each case (also in the shuffle handler above)

Reply:
will change! reviewer is always right!